controller/eventQuestionController.js

```diff
@@ -0,0 +1,291 @@
+const HttpStatus = require('http-status-codes');
+const Mongoose = require('mongoose');
+
+const EventQuestion = require('../model/eventQuestionModel');
+const Event = require('../model/eventModel');
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const eventQuestion = new EventQuestion(req.body);
+
```

Comment:
Shouldn't you first check if the user that is sending the question sending it for an event that is created by his own company? It looks to me that this way a user could theoretically send a question for an event out of his company.

Reply:
Will do after adding companyID to eventQuestion model. As it is now, there is too much nested code, and the code gets far too complicated without any reason.

Reply:
Finish this and we can merge.